notebooks/non-sequential.ipynb

```diff
@@ -0,0 +1,453 @@
+{
```

Comment:
<p>I like the image of the graphical pipeline a lot. I appreciate that you included both transformations and inverse transformations in the example. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/sktime/sktime-tutorial-pydata-global-2022/pull/3/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>